element_array_ephys/probe.py

```diff
@@ -64,16 +67,16 @@ class Electrode(dj.Part):
             x_coord (float): x-coordinate of the electrode within the probe in micrometers.
             y_coord (float): y-coordinate of the electrode within the probe in micrometers.
         """
-        
+
         definition = """
         -> master
         electrode: int       # electrode index, starts at 0
         ---
         shank: int           # shank index, starts at 0, advance left to right
         shank_col: int       # column index, starts at 0, advance left to right
         shank_row: int       # row index, starts at 0, advance tip to tail
-        x_coord=NULL: float  # (um) x coordinate of the electrode within the probe, (0, 0) is the bottom left corner of the probe
-        y_coord=NULL: float  # (um) y coordinate of the electrode within the probe, (0, 0) is the bottom left corner of the probe
+        x_coord=NULL: float  # (μm) x coordinate of the electrode within the probe, (0, 0) is the bottom left corner of the probe
+        y_coord=NULL: float  # (μm) y coordinate of the electrode within the probe, (0, 0) is the bottom left corner of the probe
```

Comment:
hm.. does mysql even support non-ascii characters in comments?

Reply:
I think so.. tables are created without any errors.

Reply:
What do you get from `ProbeType.Electrode.describe()`?